### DISCUSSION TITLE
Show Customers Reward Points in Pos

### DISCUSSION THREAD

**Discussion by Tex1000** [Blair2004/NexoPOS#306] [Ideas]
Pls fetch the customers rewards points in the pos too, just as we can see customers coupons. it is very important, right now we have to always go to the dashboard to see the customers points. It’s not convenient at all. All important data should be viewable on the pos when we open a customers profile

**Comment by Blair2004** [comment]
A customer might have points from many reward systems. So, we cannot display a total point that a customer. This is mainly because the points displayed will be aggregated from all reward systems the customer is involved. We might alternatively display a list of reward systems the customer is in with the point and target.

**Comment by Tex1000** [comment]
I just updated and my site is broken. It shows an error in dispatcher.php file.   Public function subscribe

**Comment by Tex1000** [comment]
![Uploading 5B546505-2E8C-44ED-957A-8DFAB25A4D06.jpeg…]()
You can see the picture here

**Comment by Blair2004** [comment]
Please if you have a new issue you're facing do not misc it with a feature request.

**Comment by Tex1000** [comment]
![C165756C-49D9-4300-99DB-360291A6B48E](https://user-images.githubusercontent.com/89839039/132955275-b3c55dbc-0480-4f1d-ab6b-ec93bea4463f.jpeg)


> A customer might have points from many reward systems. So, we cannot display a total point that a customer. This is mainly because the points displayed will be aggregated from all reward systems the customer is involved. We might alternatively display a list of reward systems the customer is in with the point and target.

In my opinion bro, it’s okay, customers ask how many reward points they have, so it’s okay to show a total of all points. We can then add a tab to show the list you mentioned for those who may want to know that, but having a display of all rewards points for each customer is important. It will allow us some space to be more creative with the system, with that data we can do example decide to give the customer some cash back base on his total points.  I updated the update u released last night and now my site is broken. It shows the error I posted. Can’t have access to any page. Kindly check the update u released and release another to fix this bug

**Comment by Blair2004** [comment]
Hi,
NexoPOS 4.5.2 has been released. Your modules might not be compatible. If you have installed some. Otherwise we'll need to access your system (on monday).

**Comment by Tex1000** [comment]
> Hi,
> NexoPOS 4.5.2 has been released. Your modules might not be compatible. If you have installed some. Otherwise we'll need to access your system (on monday).

Pls come on Skype so I can check my system as you usually do. My server details are with u on Skype
